client/termdb/TermdbVocab.js

```diff
@@ -477,33 +477,40 @@ export class TermdbVocab extends Vocab {
 		return await dofetch3('/termdb', { body })
 	}
 
-	async getterm(termid, dslabel = null, genome = null) {
-		if (!termid) throw 'getterm: termid missing'
+	async getTerms(ids, dslabel = null, genome = null) {
+		if (!ids) throw 'getTerms: ids missing'
 		if (this && this.state && this.state.vocab) {
 			if (this.state.vocab.dslabel) dslabel = this.state.vocab.dslabel
 			if (this.state.vocab.genome) genome = this.state.vocab.genome
 		}
```

Comment:
@airenzp The logic above cancels the overrides if supplied as arguments. This should be
```js
async getTerms(ids: string[], _dslabel = null, _genome = null) {
   if (!ids) throw 'getTerms: ids missing'
  const dslabel = _dslabel || this.state.vocab?.dslabel // this.vocab is guaranteed to exist?
  const genome = _genome || this.state.vocab?.genome // this.vocab is guaranteed to exist?
```